### PR TITLE
HyperShift QE: Remove 4.13 and add 4.14 for the AWS part.

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by heliubj18** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-hypershift-guest-f7

**Comment by heliubj18** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-hypershift-guest-f7

**Comment by heliubj18** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-rosa-sts-hypershift-guest-f7

**Comment by LiangquanLi930** [comment]
@liangxia  Could you help to approve this PR? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -3,34 +3,6 @@ base_images:
     name: "4.13"
     namespace: ocp
     tag: ansible
-  assisted-image-service:
-    name: ocm-2.8
-    namespace: edge-infrastructure
-    tag: assisted-image-service
-  assisted-installer:
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:10]: Could you double confirm that only hypershift and no other tests use those images ?

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:10]: yes, i can confirm it, these images all support hypershift-agent workflow

**Pull request merged**